wagtail/images/tests/test_admin_views.py

```diff
@@ -2229,6 +2236,35 @@ def filter_images(images, request):
         self.assertEqual(len(response.context["results"]), 1)
         self.assertEqual(response.context["results"][0], image)
 
+    def test_list_mode(self):
```

Comment:
#13382 didn't add any tests for loading the chooser view in list mode, so I'm adding one here